lib/types/api.types.ts

```diff
@@ -262,7 +279,11 @@ export type FieldAppSDK = BaseAppSDK &
     window: WindowAPI
   }
 
-export type DialogAppSDK = Omit<BaseAppSDK, 'ids'> & {
+export type DialogAppSDK<
+  InstallationParameters extends KeyValueMap = KeyValueMap,
+  InstanceParameters extends KeyValueMap = KeyValueMap,
+  InvocationParameters extends SerializedJSONValue = SerializedJSONValue
+> = Omit<BaseAppSDK<InstallationParameters, InstanceParameters, InvocationParameters>, 'ids'> & {
```

Comment:
Dialog location has no instance params

```suggestion
export type DialogAppSDK<
  InstallationParameters extends KeyValueMap = KeyValueMap,
  InvocationParameters extends SerializedJSONValue = SerializedJSONValue
> = Omit<BaseAppSDK<InstallationParameters, never, InvocationParameters>, 'ids'> & {```